devtools/conda-recipe/meta.yaml

```diff
@@ -28,7 +29,7 @@ requirements:
     - six
     - openmm
     - parmed
-
+    - tqdm
```

Comment:
I think this is the dependency that is breaking python 3.4 tests. Do we need it?